libcontainer/cgroups/systemd/v2.go

```diff
@@ -302,7 +302,8 @@ func (m *unifiedManager) Destroy() error {
 	}
 
 	// XXX this is probably not needed, systemd should handle it
-	err := os.Remove(m.path)
+	logrus.Warnf("try to delete %s", m.path)
```

Comment:
Debugf

Reply:
Sorry for my mistake, I have modified it.